tests/unit/mock/config/compliance/config_section_not_parsed/mikrotik_routeros/routeros_feature.py

```diff
@@ -0,0 +1,6 @@
+features = [
```

Comment:
Always like to have an example of a banner being parsed.

Reply:
Hey Ken! The comments at the top of the config files, is that considered a banner here, not just comments? There is a `system note` command, but it just outputs like a normal str config line. 

```[admin@R2] > export terse
# mar/22/2023 17:44:57 by RouterOS 6.48.5
# software id =
#
#
#
/interface wireless security-profiles set [ find default=yes ] supplicant-identity=MikroTik
/ip address add address=172.31.255.30/30 interface=ether1 network=172.31.255.28
/ip dhcp-client add disabled=no interface=ether1
/system identity set name=R2
/system note set note="SOME CRAZY NOTE!"
```

Reply:
ok, perhaps simpleton of me to presume there is a banner :) 

I do think it would be great to have a few options for "note", like how does it handle multi-line (is that supported), special characters, can you have a double quote in there, etc.. Think of ways to break it

Reply:
@itdependsnetworks I added to the `parsing` tests to capture the lines that are children of the `/system note set note=..` line. This is the exact input that went into the device is below. If a user inputs a string without adding `\n` the config export will handle it and add it for you, same with special chars.  Device will just escape them when exporting config.

```bash
This is a \"System Note\" for a Mikrotik router.\n\
It includes double quotes (\") and special characters such as:\n\
@, #, $, %, ^, &, *, (, ), _, +, [, ], {, }, |, ;, ',', ., /, <, >, and ?.\n\
\n\
Remember to escape any special characters with a backslash (\\) when necessary.\n\
\n\
This is a multiline note with several lines of text.
```

and when doing a `terse` config export, the following output appears ->

```bash
[admin@ag1.123site.nwk.nj] /system note> export terse
# apr/01/2023 17:16:02 by RouterOS 6.48.5
# software id =
#
#
#
/system note set note="This is a \\\"System Note\\\" for a Mikrotik router.\\n\\\
    \nIt includes double quotes (\\\") and special characters such as:\\n\\\
    \n@, #, \$, %, ^, &, *, (, ), _, +, [, ], {, }, |, ;, ',', ., /, <, >, and \?.\\n\\\
    \n\\n\\\
    \nRemember to escape any special characters with a backslash (\\\\) when necessary.\\n\\\
    \n\\n\\\
    \nThis is a multiline note with several lines of text.\\n\\\
    \nWow, what a great example of a note to ensure proper parsing by NetUtils!"
[admin@ag1.123site.nwk.nj] /system note>
```

hope this helps and captures what you asked for, LMK.